relay/views/subscriptions.view.lkml

```diff
@@ -0,0 +1,440 @@
+include: "//looker-hub/relay/views/subscriptions.view"
+  view: +subscriptions {
+
+  dimension: plan_id {
+    hidden: yes
+    }
+
+
+    dimension: subscription_id {
+      hidden: yes
+      primary_key: yes
+    }
+
+    dimension: original_subscription_id {
+      hidden: yes
+    }
+
+    dimension: country {
+      hidden: yes
+    }
+
+    dimension: country_name {
+      description: "Add placeholder string for null values.  This is to allow selection of Null values in country name checkbox filters in dashboards"
+      sql: COALESCE(${TABLE}.country_name, 'Null') ;;
+      type: string
+    }
+
+    dimension: cancel_at_period_end {
+      description: "Indicates if auto-renewal was cancelled and subscription is to be cancelled at period end"
+      type: yesno
+      sql: ${TABLE}.cancel_at_period_end;;
+    }
+
+    dimension: is_ended {
+      description: "Indicates if subscription has ended"
+      type: yesno
+      sql: ${TABLE}.ended_at IS NOT NULL;;
+    }
+
+    dimension: plan_type {
+      description: "Indicates the plan type (bundle, email month, phone year etc)"
+      type: string
+      sql:  CASE
+            WHEN ${product_name} ="Mozilla VPN & Firefox Relay" THEN CONCAT("bundle","_",${plan_interval})
+            ELSE
+              (CASE
+                WHEN (${plan_amount} > 200 AND ${plan_interval} = "month") OR (${plan_amount}  > 1200 AND ${plan_interval} = "year") THEN CONCAT("phone","_",plan_interval)
+                ELSE CONCAT("email","_",${plan_interval})
+              END)
+            END;;
+    }
```

Comment:
Changing the plan_type definition
```suggestion
      sql:  CONCAT(
            CASE
              WHEN ${product_name} LIKE "%Relay%" THEN "bundle"
              WHEN (${plan_interval} = "month" AND ${plan_amount} > 400)
                OR (${plan_interval} = "year" AND ${plan_amount} > 4000)
                THEN "phone"
              ELSE "email"
            END,
            "_", ${plan_interval_count},
            "_", ${plan_interval}
);;
```

Reply:
Because this is a Relay view, the bundle logic should check for `${product_name} LIKE "%VPN%" (not `LIKE "%Relay%"`, which would match many normal Relay plans).

Reply:
Duh..